pulumitest/opttest/opttest.go

```diff
@@ -146,11 +171,13 @@ func (o *Options) Copy() *Options {
 }
 
 var defaultConfigPassphrase string = "correct horse battery staple"
+var defaultStackName string = "test"
 
 // Defaults sets all options back to their defaults.
 // This can be useful when using CopyToTempDir or Convert but not wanting to inherit any options from the previous PulumiTest.
 func Defaults() Option {
 	return optionFunc(func(o *Options) {
+		o.StackName = defaultStackName
```

Comment:
Did you leave out `SkipInstall` and `SkipStackTrace` because their default is `false` anyway? But then why is `TestInPlace` here?